packages/vscode-extension/src/debug/commonUtils.ts

```diff
@@ -394,3 +395,21 @@ export class Step {
     return `(${this.currentStep++}/${this.totalSteps})`;
   }
 }
+
+export async function getV3TeamsAppId(projectPath: string, env: string): Promise<string> {
+  const result = await envUtil.readEnv(projectPath, env);
+  if (result.isErr()) {
+    throw result.error;
+  }
+
+  const teamsAppId = result.value.TEAMS_APP_ID;
```

Comment:
In use existing scenario, user may use other environment variable to set teams app id. What's our attitude to this case? @kuojianlu @swatDong  @jidddddd

Reply:
For existing scenario, user can directly use the raw teams app id instead of the placeholder in launch url.
@swatDong Any other ideas?

Reply:
Use `env:TEAMS_APP_ID` as placeholder?
E.g.,
- `local:TEAMS_APP_ID` means `TEAMS_APP_ID` in *teamsfx/.env.local*
- `dev:TEAMS_APP_ID` means `TEAMS_APP_ID` in *teamsfx/.env.dev*
- `TEAMS_APP_ID` means ask for env first, then from *teamsfx/.env.<env>*
- `my-dev-1:MY_TEAMS_APP_ID` means `MY_TEAMS_APP_ID` in *teamsfx/.env.my-dev-1*

Anyway, consider improve after happy path.

Reply:
BTW, do we have env selection when local debug? If yes maybe just use `TEAMS_APP_ID` as placeholder (no need to add `dev:`)?

Reply:
Currently we have (4.1.0), but only for remote. For local better to not ask, that's why we need `local:`.
As if we support `local:`, user can use any `<env>`, e.g., `dev:`.

Reply:
That's a tradeoff:
- if we use `dev:TEAMS_APP_ID` by default, the first remote launch does not require env selection, but user needs to change that for another env.
- if we use `TEAMS_APP_ID` by default, all remote launches require env selection.

I'll discuss with PM to determine the final UX, but the underlying implementation is the same.